tests/utils/test_containers.py

```diff
@@ -99,6 +99,10 @@ def test_split_unpack():
     x = torch.Tensor([1])
     y = torch.Tensor([2])
 
+    tensors, packed_non_tensors = split_non_tensors(x)
```

Comment:
nice addition to the test

Reply:
i pulled this into a PR: https://github.com/facebookresearch/fairscale/pull/1000